src/commands/report-failure.ts

```diff
@@ -15,8 +15,7 @@ function getIssueTitle(runId: string): string {
 }
 
 interface CreateIssueArgs {
-  // github: Octokit; // makes ts sad for some reason
```

Comment:
Big thanks to @jamescdavis for fixing this for us upstream (in https://github.com/octokit/rest.js/pull/1650).